cudapoa/src/cudapoa_nw_banded.cuh

```diff
@@ -297,46 +299,39 @@ __device__
 
             while (__any_sync(FULL_MASK, loop))
             {
-
-                // To increase instruction level parallelism, we compute the scores
```

Comment:
and can you add the comment here that this depends on the profile of the reads - more accurate/short read data may benefit from the ILP vs the less accurate read data benefits from the reduced while loop count?

Reply:
done